.github/workflows/trivy.yml

```diff
@@ -21,6 +21,8 @@ jobs:
 
       # fetch SBT package
       - uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
```

Comment:
Trivy checks were failing because the build step was failing because CRC32C was introduced in Java 9 and Trivy was defaulting to Java 8.